### PR TITLE
include id, type, service and subservice in ctxt for evaluate expresion in command

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue related https://github.com/telefonicaid/iotagent-json/issues/634

### CODE REVIEW

**Comment by AlvaroVega** [comment]
Check (and possible apply expression to poll commands)

**Comment by fgalan** [comment]
Tiwn PR in IOTA-UL needed?

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Add: include device ID, TYpe, Service and SubService in context to expression push command
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: Typo:

```suggestion
- Add: include device ID, Type, Service and SubService in context to expression push command
```

In addition, should issue https://github.com/telefonicaid/iotagent-json/issues/634 be mentioned in this line?

**Comment by fgalan** [comment]
> > Tiwn PR in IOTA-UL needed?
> 
> Twin PR for iota-ul is comming: [telefonicaid/iotagent-ul#565](https://github.com/telefonicaid/iotagent-ul/pull/565)

Let's wait to have that one in no-WIP state so we can merge both at the same time.

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**